src/vm/zapsig.cpp

```diff
@@ -262,6 +262,10 @@ BOOL ZapSig::GetSignatureForTypeHandle(TypeHandle      handle,
 
```

Comment:
Is this signature encoder used by IBC as well? Is this going to break IBC format?

Reply:
It id  likely that IbcMerge tool needs an update to track this change